osclib/accept_command.py

```diff
@@ -32,6 +32,39 @@ def find_new_requests(self, project):
             rqs.append({'id': int(rq.get('id')), 'packages': pkgs})
         return rqs
 
+    def virtual_accept_request_has_no_binary(self, project, package):
+        pkgs = []
+        filelist = self.api.get_filelist_for_package(pkgname=package, project=self.api.project, expand='1', extension='spec')
+        for spec in filelist:
+            pkgs.append(spec[:-5])
+
+        for pkg in pkgs:
+            query = {'view': 'binarylist', 'package': pkg, 'multibuild': '1'}
+            pkg_binarylist = ET.parse(http_GET(self.api.makeurl(['build', project, '_result'], query=query))).getroot()
+            for binary in pkg_binarylist.findall('./result/binarylist/binary'):
+                return False
+
+        return True
+
+    def find_virtual_accepted_requests(self, project):
```

Comment:
Rather than duplicate all the dom walking code from `find_new_requests()` it would seem to make more sense to rename that method to somethign like `find_requests()` and add a parameter to indicate looking for virtual delete requests.